cfi/lib/src/cfi_counter.rs

```diff
@@ -84,7 +84,7 @@ impl CfiCounter {
     /// Reset counter
     #[inline(always)]
     pub fn reset(trng: &mut caliptra_drivers::Trng) {
-        prng().mix_entropy_from_trng(trng);
```

Comment:
We still need this part; we need to mix the entropy from all three calls to this function.